static/app/components/platformPicker.tsx

```diff
@@ -87,7 +87,18 @@ class PlatformPicker extends Component<PlatformPickerProps, State> {
       return currentCategory?.platforms?.has(platform.id);
     };
 
-    const filtered = selectablePlatforms
+    // temporary replacement of selectablePlatforms while `nintendo` is behind feature flag
+    const tempSelectablePlatforms = selectablePlatforms;
+    if (this.props.organization?.features.includes('selectable-nintendo-platform')) {
+      const nintendo = platforms.find(p => p.id === 'nintendo');
+      if (nintendo) {
+        if (!tempSelectablePlatforms.includes(nintendo)) {
+          tempSelectablePlatforms.push(nintendo);
+        }
+      }
+    }
+
+    const filtered = tempSelectablePlatforms
```

Comment:
is this the best place to have this? I imagine we might want to push this selectablePlatforms list to a lot of components, maybe even in the future. Any chance we can do this closer to the source of truth?

Reply:
It was there in the previous commit and Arthur in comment above asked me to move it here. I think it makes sense for now at least to have less visibility until Nintendo provides public sdk and documentation.

Reply:
oh haha, sorry about that